protos/proto/aptos/transaction/v1/transaction.proto

```diff
@@ -518,6 +518,7 @@ message AnyPublicKey {
     TYPE_SECP256K1_ECDSA = 2;
     TYPE_SECP256R1_ECDSA = 3;
     TYPE_KEYLESS = 4;
+    TYPE_FEDERATED_KEYLESS = 5;
```

Comment:
let's get stamp from @bowenyang007 / @larry-aptos for this?

Reply:
Yes, would be great. I updated the indexer GRPC stuff. (I believe.)

Reply:
looks good.